robot/EDA/resources/EDA.py

```diff
@@ -418,3 +418,33 @@ def verify_dropdown_field_status(self, **kwargs):
             expected_value = bool(expected_value == "disabled")
             if not str(expected_value).lower() == str(actual_value).lower() :
                 raise Exception (f"Drop down field {field} status is {actual_value} instead of {expected_value}")
+
+    def verify_action_button_status(self, **kwargs):
+        """ Verify the action button is disabled/enabled for the user
+            we have to pass the name of the button and the expected status
+            of the action button as either enabled or disabled
+        """
+        for button,expected_value in kwargs.items():
+            locator = eda_lex_locators["eda_settings"]["run_action"].format(button)
+            self.selenium.page_should_contain_element(locator)
+            self.selenium.wait_until_element_is_visible(locator,
+                                                error= f"Element '{button}' button is not displayed for the user")
+            time.sleep(1)
+            actual_value = self.selenium.get_webelement(locator).get_attribute("disabled")
+            print(actual_value)
```

Comment:
Delete this if this was just for local debugging. But if you need it, then update it to `self.builtin.log("Actual value: " + str(actual_value))`

Reply:
Yep. It was for debugging purpose. I will remove it